src/platform/plugins/shared/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -146,6 +146,7 @@ export interface QueryBarTopRowProps<QT extends Query | AggregateQuery = Query>
   onRefreshChange?: (options: { isPaused: boolean; refreshInterval: number }) => void;
   onSubmit: (payload: { dateRange: TimeRange; query?: Query | QT }) => void;
   onCancel?: () => void;
+  onDraftChange?: (draft: UnifiedSearchDraft | null) => void;
```

Comment:
Curious why we use `null` here instead of `undefined` like the other callbacks?

Reply:
Updated to `undefined`